packages/backend/src/execution/executors/PgExecutor/PgExecutor.test.ts

```diff
@@ -83,6 +83,27 @@ describe('PgExecutor', () => {
         ]);
     });
 
+    const q3 = from('actor')
+        .columns('actor_id', 'first_name', 'last_name')
+        .where({ actor_id: 1 })
+        .one();
+
+    it(`Actor table SynthQL query executes to 
+        expected result with SQL to prepend included`, async () => {
+        const result = await executor.execute(q3, {
+            ...executeProps,
+            prependSql: 'BEGIN',
```

Comment:
this test doesn't make sense 🤔 your'e just opening the connection but never closing it